pages/userSettings/userSettings.js

```diff
@@ -37,34 +37,69 @@ router.post('/', (req, res, next) => {
             if (ERR(err, next)) return;
             res.redirect(req.originalUrl);
         });
-    } else {
+    }
+    else if(req.body.__action === 'update_settings'){
```

Comment:
Please follow code style of other code (e.g., spaces around keywords and parens).